### ISSUE TITLE
Issues while using between operator

### ISSUE THREAD

**Issue by viramjainfyno** [react-querybuilder/react-querybuilder#726]
### React Query Builder version

7.4.4

### Description

When I try to use between operator for a text field, if the text field contains a comma, the generated SQL is fine but when I try to import the SQL, the UI gets messed up.
Please refer to the attached video

https://github.com/react-querybuilder/react-querybuilder/assets/150130308/bd84e072-8af8-4470-a75b-6b33b1f1965b

### Reproduction

NA

### Expected behavior

I should be able to switch between operators seamlessly

**Comment by jakeboone02** [comment]
That's a good catch. You can sort of work around it today with the `listsAsArrays` option:

```ts
parseSQL("firstName between 'Vi,ram' and 'Jain'", { listsAsArrays: true })
```
yields
```json
{
  "combinator": "and",
  "rules": [
    {
      "field": "firstName",
      "operator": "between",
      "value": ["Vi,ram", "Jain"]
    }
  ]
}
```

However that doesn't really address the bug, so I've resolved it in #716.

**Comment by jakeboone02** [comment]
Fix for this was included in [v7.5.0](https://github.com/react-querybuilder/react-querybuilder/releases/tag/v7.5.0).
